common/util.go

```diff
@@ -179,12 +179,15 @@ func PrettyPrintJSONObject(o interface{}) {
 		b, err = json.MarshalIndent(o, "", "  ")
 	}
 
+	w := c.App.Writer
+
 	if err != nil {
-		fmt.Printf("Error when try to print pretty: %v", err)
-		fmt.Println(o)
+		fmt.Fprintf(w, "Error when try to print pretty: %v", err)
```

Comment:
All `fmt.Printf` should be `fmt.Fprintf`. No?

Reply:
yes! i'll send separately to not mix